src/helpers/utils.ts

```diff
@@ -247,3 +248,16 @@ async function updateWalletConnectWhitelist(
 
   return true;
 }
+
+export function getSpaceController(space: string, network = NETWORK) {
```

Comment:
I think it is better to use `NETWORK` directly inside this function instead of getting same thing from params

But not too important